src/RelayClient.ts

```diff
@@ -364,10 +370,14 @@ class RelayClient extends EnvelopingEventEmitter {
   }
 
   public async relayTransaction(
-    envelopingRequest: UserDefinedEnvelopingRequest
+    envelopingRequest: UserDefinedEnvelopingRequest,
+    options?: RelayTxOptions
   ): Promise<Transaction> {
+    const signerWallet = options?.signerWallet;
+
     const { envelopingTx, activeRelay } = await this._getHubEnvelopingTx(
-      envelopingRequest
+      envelopingRequest,
+      signerWallet
```

Comment:
NIT: I wouldn't use an additional variable to store it.
```suggestion
      envelopingRequest,
      options?.signerWallet
```